go/grpcwebproxy/main.go

```diff
@@ -151,7 +151,7 @@ func buildGrpcProxyServer(logger *logrus.Entry) *grpc.Server {
 		md, _ := metadata.FromIncomingContext(ctx)
 		outCtx, _ := context.WithCancel(ctx)
 		mdCopy := md.Copy()
-		delete(mdCopy, "user-agent")
```

Comment:
I think we still want to delete this header, no?

Reply:
Okay, I restored it! Do you know why it's being deleted?

Reply:
Nope!